chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -1675,6 +1675,37 @@ parameter_types! {
     pub const FreeCollatorAssignmentCredits: u32 = FreeBlockProductionCredits::get()/EpochDurationInBlocks::get();
 }
 
+pub struct EnsureParachainManager;
```

Comment:
Very likely this should be both on dancelight and stagelight. And being that the case, I would probably this somewhere in common. Also tests